docs/contents/services.md

```diff
@@ -1,204 +1,223 @@
-| Amazon [DynamoDB](#amazon-dynamodb) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/issues/2060) |
-| Amazon [ECR](#amazon-ecr) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/issues/208) |
-| Amazon [EFS](#amazon-efs) | [`PROPOSED`](https://github.com/aws/aws-controllers-k8s/issues/328) |
-| Amazon [EKS](#amazon-eks) | [`PLANNED`](https://github.com/aws/aws-controllers-k8s/issues/16) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/7)
-| Amazon [ElastiCache](#amazon-elasticache) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/issues/240) | [`BETA`](https://github.com/aws/aws-controllers-k8s/milestone/9)
-| Amazon [Elasticsearch](#amazon-elasticsearch) | [`PROPOSED`](https://github.com/aws/aws-controllers-k8s/issues/503) |
-| Amazon [EC2 VPC](#amazon-ec2-vpc) | [`PROPOSED`](https://github.com/aws/aws-controllers-k8s/issues/489) |
-| AWS [IAM](#aws-iam) | [`PROPOSED`](https://github.com/aws/aws-controllers-k8s/issues/222) |
-| AWS [Lambda](#aws-lambda) | [`BUILD`](https://github.com/aws/aws-controllers-k8s/issues/238) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/10)
-| AWS [Kinesis](#aws-kinesis) | [`PROPOSED`](https://github.com/aws/aws-controllers-k8s/issues/235) |
-| Amazon [KMS](#amazon-kms) | [`BUILD`](https://github.com/aws/aws-controllers-k8s/issues/491) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/18)
-| Amazon [MQ](#amazon-mq) | [`BUILD`](https://github.com/aws/aws-controllers-k8s/issues/390) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/12)
-| Amazon [MSK](#amazon-msk) | [`PLANNED`](https://github.com/aws/aws-controllers-k8s/issues/348) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/13)
-| Amazon [RDS](#amazon-rds) | [`BUILD`](https://github.com/aws/aws-controllers-k8s/issues/237) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/8)
-| Amazon [Route53](#amazon-route53) | [`PROPOSED`](https://github.com/aws/aws-controllers-k8s/issues/480) |
-| Amazon [SageMaker](#amazon-sagemaker) | [`BUILD`](https://github.com/aws/aws-controllers-k8s/issues/385) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/11)
-| Amazon [SNS](#amazon-sns) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/issues/202) |
-| Amazon [SQS](#amazon-sqs) | [`BUILD`](https://github.com/aws/aws-controllers-k8s/issues/205) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/milestone/6)
-| AWS [Step Functions](#aws-step-functions) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/issues/239) |
-| Amazon [S3](#amazon-s3) | [`DEVELOPER PREVIEW`](https://github.com/aws/aws-controllers-k8s/issues/204) |
+| Amazon [ACM](#amazon-acm) | [`PROPOSED`](https://github.com/aws-controllers-k8s/community/issues/482) | |
```

Comment:
Is it possible to show this as a table view?

Reply:
This is a table view, just above you see the Markdown format:

https://aws-controllers-k8s.github.io/community/services/